tests/Neo.Plugins.Storage.Tests/StoreTest.cs

```diff
@@ -49,35 +58,143 @@ public void TestMemory()
         [TestMethod]
         public void TestLevelDb()
         {
-            using var plugin = new LevelDBStore();
-            TestPersistenceDelete(plugin.GetStore(path_leveldb));
+            ReSetStore();
```

Comment:
Already donde in Initialize, isn't it?

Reply:
Well, yes, but since there is an if check, should be fine.